src/analytics.js

```diff
@@ -56,3 +57,28 @@ export function triggerAnalyticsEvent(nodeOrDoc, eventType, opt_vars) {
     analytics.triggerEvent(eventType, opt_vars);
   });
 }
+
+/**
+ * Helper method to create analytics element for specific root.
+ * @param {BaseElement} baseInstance
+ * @param {!JSONType} config
+ */
+export function createAnalyticsForExtension(baseInstance, config) {
+  // Note: Require ampdoc include analytics script
+  analyticsForDocOrNull(baseInstance.element).then(analytics => {
+    if (!analytics) {
+      return;
+    }
+    // Create analytics element;
+    const analyticsElem =
+        baseInstance.win.document.createElement('amp-analytics');
+    const scriptElem = createElementWithAttributes(baseInstance.win.document,
+        'script', {
+          'type': 'application/json',
+        });
+    scriptElem.textContent = JSON.stringify(config);
```

Comment:
`scoped` is just a documented fact that the amp-analytics was inserted by us. E.g. when we debug things, this should help. In the future we can derive some features from this as well (very likely).

The JSON thing - is just an implementation to avoid excessive re-parsing.